src/ResourceManager/DevSpaces/Commands.DevSpaces.Test/packages.config

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.1.3" targetFramework="net452" />
+  <package id="Microsoft.Azure.Common" version="2.1.4" targetFramework="net452" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.DevSpaces" version="1.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.ResourceManager" version="1.6.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.7.0" targetFramework="net452" />
+  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net452" />
+  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net452" />
+  <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net452" />
+  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net452" />
+  <package id="Microsoft.Net.Http" version="2.2.22" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```

Comment:
remove clientruntime and clientruntime.azure

Reply:
Done